client/app/models/milestone/constants.js

```diff
@@ -48,6 +48,10 @@ export const MAYORAL_VOTE = 'a83beec4-dad0-e711-8116-1458d04e2fb8';
 
 export const FINAL_LETTER_SENT = 'aa3beec4-dad0-e711-8116-1458d04e2fb8';
 
+export const REVIEW_FILED_LAND_USE_APPLICATION = '6c3beec4-dad0-e711-8116-1458d04e2fb8';
+
+export const ENVIRONMENTAL_ASSESSMENT_STATEMENT_FILED = '723beec4-dad0-e711-8116-1458d04e2fb8';
+
```

Comment:
This is the main event — define these constants, import them in the milestone model, and add them to the groups in the computed property. Makes sense.